runtimes/eden/src/xcm_config.rs

```diff
@@ -151,3 +161,72 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+pub struct AccountIdToMultiLocation;
+impl Convert<AccountId, MultiLocation> for AccountIdToMultiLocation {
+	fn convert(account: AccountId) -> MultiLocation {
+		X1(AccountId32 {
+			network: NetworkId::Any,
+			id: account.into(),
+		})
+		.into()
+	}
+}
+parameter_types! {
+	pub const BaseXcmWeight: XcmWeight = 100_000_000; // TODO: update based on the results of https://github.com/NodleCode/chain-workspace/issues/259
+	pub const MaxAssetsForTransfer: usize = 2;
+}
+parameter_types! {
+	pub SelfLocation: MultiLocation = MultiLocation::here();
+}
+parameter_type_with_key! {
+	pub ParachainMinFee: |_location: MultiLocation| -> Option<u128> {
+		Some(u128::MAX)
```

Comment:
```suggestion
		None
```

Like this?

Reply:
Yup let's do this. I'm going to commit the suggestion.